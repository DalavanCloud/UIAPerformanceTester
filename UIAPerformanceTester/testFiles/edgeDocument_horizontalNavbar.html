<html>
<head>
<title>UIAPerformanceTester:horizontal Navbar</title>
<style type="text/css">
nav li {
display:inline;
}
</style>
</head>
<body>
<header role="banner">
<h1>Horizontal Navbar</h1>
<nav>
<ul>
<li><a href="#">Home</a></li>
<li><a href="#">About</a></li>
<li><a href="#">Download</a></li>
<li><a href="#">News</a></li>
<li><a href="#">Services</a></li>
<li><a href="#">Shop</a></li>
<li><a href="#">Support us</a></li>
</ul>
</nav>
</header>
<main role="main">
<p>content</p>
</main>
</body>
</html>