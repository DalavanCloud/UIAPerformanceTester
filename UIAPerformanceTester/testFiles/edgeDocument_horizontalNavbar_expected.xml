<root>
<UIAElement controlType="pane" name="Horizontal Navbar" >
...
<UIAElement controlType="header" name="" >
...
<UIAElement controlType="navigation" name="" >
...
<UIAElement controlType="list" name="" >
...
<UIAElement controlType="list item" name="Home" >
<UIAElement controlType="link" name="Home" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
Home
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 
</textRange>
<UIAElement controlType="list item" name="About" >
<UIAElement controlType="link" name="About" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
About
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 
</textRange>
<UIAElement controlType="list item" name="Download" >
<UIAElement controlType="link" name="Download" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
Download
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 
</textRange>
<UIAElement controlType="list item" name="News" >
<UIAElement controlType="link" name="News" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
News
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 
</textRange>
<UIAElement controlType="list item" name="Services" >
<UIAElement controlType="link" name="Services" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
Services
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 
</textRange>
<UIAElement controlType="list item" name="Shop" >
<UIAElement controlType="link" name="Shop" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
Shop
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 
</textRange>
<UIAElement controlType="list item" name="Support us" >
<UIAElement controlType="link" name="Support us" >
<textRange fontName="Times New Roman" fontSize="12.000000" >
Support us
</textRange>
</UIAElement>
</UIAElement>
<textRange fontName="Times New Roman" fontSize="12.000000" >
 

</textRange>
</UIAElement>
...
</UIAElement>
...
</UIAElement>
...
</UIAElement>
</root>
